org.adempiere.base/src/org/compiere/model/MInventoryLine.java

```diff
@@ -289,6 +289,10 @@ protected boolean beforeSave (boolean newRecord)
 		MDocType dt = MDocType.get(getCtx(), getParent().getC_DocType_ID());
 		String docSubTypeInv = dt.getDocSubTypeInv();
 
+		//	Set AD_Org to parent if not charge

```

Comment:
Moving these lines above seems useless and potentially dangerous - in line 330 the charge is being set to zero, so in case that line is hit this code movement will miss the change on ad_org